### DISCUSSION TITLE
Set permanent password inside strategy

### DISCUSSION THREAD

**Discussion by imello86** [rustdesk/rustdesk-server-pro#185] [General]
**Is your feature request related to a problem? Please describe.**
Massive deploy on system like android, set permanent password is too difficult, especially if you use a strong password.
Could be useful for other systems, in order to change password when needed, without deploy again via script the client.

**Describe the solution you'd like**
Set permanent password (when selected) inside strategy configuration.

**Comment by rustdesk** [comment]
Many users asked for this, but I am still hesitating for security concern. Do you know any other vendors provide this, and how?

**Reply by pdifeodfware** [reply]
> Do you know any other vendors provide this, and how?

Perhaps not the best for security. But for example TeamViewer allows you to create a version of QuickSupport with a password that you pass on when creating your customised package. This is very helpful. Not all end users are familiar with this type of software.

**Reply by rustdesk** [reply]
This is ok for me, but putting in strategy is not a good idea.

**Reply by imello86** [reply]
Create a QuickSupport app is a valid alternative, but the app must be installed as a service, not only on demand.

The main problem is on Android/iOS devices, because I can't deploy app configurated like windows or Linux systems via script, where permanent password can be set throw parameter --password.

In my case, I have to manage more than 50 Android devices.

**Reply by rustdesk** [reply]
Of course, we can understand the use case. But we can not take the risk since the other vendors do not do so.

**Reply by rustdesk** [reply]
You can change password on Android like this, it can be done with adb, I think it can also be done with some android automation tool like intune, but these tool might be not cheap.

https://github.com/rustdesk/rustdesk/issues/616#issuecomment-1837468064

**Reply by rustdesk** [reply]
I think we can put this function in custom build, but it can not be in public official release.

**Reply by imello86** [reply]
Main focus on security and practicality, but available for all (or Pro version)
